CalibTracker/SiStripQuality/test/test_makeMergedBadComponentsWithFedErr.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+function die { echo $1: status $2 ;  exit $2; }
+run="319176"
+dqmFile="/eos/cms/store/group/comm_dqm/DQMGUI_data/Run2018/ZeroBias/R0003191xx/DQM_V0001_R000319176__ZeroBias__Run2018B-PromptReco-v2__DQMIO.root"
```

Comment:
@jfernan2 how long do you reckon will these legacy files be stored on the DQM group eos area?

Reply:
In principle there is no deadline, so that address is fine for the long term, and since we are moving to the new DQM GUI which will use eos as backend (with that same address if quota gets increased, or a new address), I believe it is safe to use it.
Thanks for asking!